src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1563,6 +1563,7 @@
 
   <Node Name="BoundDeclarationPattern" Base="BoundPattern">
     <Field Name="LocalSymbol" Type="LocalSymbol" Null="disallow"/>
```

Comment:
><Field Name="LocalSymbol" Type="LocalSymbol" Null="disallow"/> [](start = 4, length = 62)

We probably should try getting rid of this property because in top level script we would want the target to be a field rather than a local.